integration/src/integration/java/pl/allegro/tech/hermes/integration/management/TopicManagementTest.java

```diff
@@ -209,4 +209,22 @@ public void shouldCreateTopicEvenIfExistsInBrokers() {
         assertThat(response).hasStatus(Response.Status.CREATED);
         Assertions.assertThat(management.topic().get(qualifiedTopicName)).isNotNull();
     }
+
+    @Test
+    public void topicCreationRollbackShouldNotDeleteTopicOnBroker() throws Throwable {
+        // given
+        String groupName = "existingTopicFromExternalBroke22r";
```

Comment:
Sry for that, now it's renamed for: "topicCreationRollbackShouldNotDeleteTopicOnBroker"